### ISSUE TITLE
Not enough documentation available on how to use cog.File as an output type.

### ISSUE THREAD

**Issue by HG2407** [replicate/cog#1488]
I want to get a snippet displaying how to use cog.File as a return (output) type of the predict function. There is not enough documentation available on the topic. Please add the documentation or (preferably) send the snippet showcasing its usage. Thanks!

**Comment by manu-sapiens** [comment]
I agree with that request. Especially since File() does not seem to take an argument, unlike what the current documentation shows!

**Comment by markphelps** [comment]
Closing this issue as stale since there has been no activity for over two years. If the documentation for using `cog.File` as a predict output type is still missing or incorrect in the current version of Cog, please feel free to reopen or create a new issue with a link to the current docs page. Thanks!
